apps/video/index.html

```diff
@@ -152,18 +167,20 @@ <h1 id="video-title"></h1>
     <!-- This element is the template of each thumbnail item -->
     <div id="thumbnail-group-header" class="hidden">
       <!--
-        <li>
-          <div class="thumbnail-group-header"></div>
-          <ul class="thumbnail-group-container"></ul>
+        <li role="presentation">
+          <div class="thumbnail-group-header" role="heading"
+               aria-level="1"></div>
+          <ul class="thumbnail-group-container" role="listbox"
+              data-l10n-id="videos-list" aria-multiselectable="true"></ul>
         </li>
       -->
     </div>
     <div id="thumbnail-template" class="hidden">
       <!--
-        <li class="thumbnail">
+        <li class="thumbnail" role="option">
           <div class="inner">
             <div class="unwatched"></div>
-            <div class="img"></div>
+            <div class="img" role="presentation"></div>
```

Comment:
I think it would be helpful to include a comment regarding the "presentation" role and why it's being used here.